Stride.BepuPhysics/Definitions/Colliders/_ColliderBase.cs

```diff
@@ -23,21 +23,30 @@ public float Mass
                 Container?.ContainerData?.TryUpdateContainer();
             }
         }
-        public Vector3 LinearOffset
+
+        /// <summary>
+        /// Local position of this collider relative to its parent
+        /// </summary>
+        [DataAlias("LinearOffset")]
+        public Vector3 PositionLocal
         {
-            get => _linearOffset;
+            get => _positionLocal;
             set
             {
-                _linearOffset = value;
+                _positionLocal = value;
                 Container?.ContainerData?.TryUpdateContainer();
             }
         }
-        public Vector3 RotationOffset
+
+        /// <summary>
+        /// Local rotation of this collider relative to its parent
+        /// </summary>
+        public Quaternion RotationLocal
```

Comment:
Changed it to quaternion to reflect how transform works, and avoid the ambiguity between euler and radians as values for vector3